packages/pigeon/platform_tests/test_plugin/ios/Classes/CoreTests.gen.swift

```diff
@@ -281,7 +281,22 @@ protocol HostIntegrationCoreApi {
   /// Returns the passed string asynchronously.
   func echoAsyncString(aString: String, completion: @escaping (String) -> Void)
   func callFlutterNoop(completion: @escaping () -> Void)
+  func callFlutterEchoAllTypes(everything: AllTypes, completion: @escaping (AllTypes) -> Void)
+  func callFlutterSendMultipleNullableTypes(aNullableBool: Bool?, aNullableInt: Int32?, aNullableString: String?, completion: @escaping (AllNullableTypes) -> Void)
```

Comment:
Same with swift

Reply:
Tracked by https://github.com/flutter/flutter/issues/41129; hopefully we'll have that fairly soon.